### PR TITLE
Add a pop-up in the evaluation

### PR DESCRIPTION
**Pull request by sofipacifico**
* Added pop-up to show a message before the evaluation finishes

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 1]
```diff
@@ -81,26 +81,30 @@
...
-            if (this.evaluation.readyForDevolution) {
-                this.evaluationFinished = true;
-            } else {
-                this.calificationFinished = true;
-            }
-            this.onSave();
+            modalViewModel.title("Finalizar evaluación");
+            modalViewModel.text("¿Desea finalizar la evaluación? Recuerde que una vez finalizada tu evaluación no podrás volver a editarla");
+            modalViewModel.buttonBackText("Cancelar");
+            modalViewModel.buttonFinalText("Finzalizar");
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: `Finzalizar` => `Finalizar`

**Comment by andresmoschini** [comment]
Looks fine

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js (hunk 1 above)
Comment (sofipacifico) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: Changed!

**Comment by joseguti84** [comment]
:+1:

**Pull request merged**